setup/nextcloud.sh

```diff
@@ -90,6 +90,24 @@ InstallNextcloud() {
 	echo "Upgrading to Nextcloud version $version"
 	echo
 
+	# set PHP version, set to earlier version if required for upgrade
+	nc_php_ver="$PHP_VER"
+	if [[ ${version} =~ ^2[0123] ]]; then
```

Comment:
Is that because older Nextcloud versions can't run on PHP v8.0 ?

Reply:
Earlier versions of nextcloud (20-23) can't run on PHP 8.1. This block is so that it uses PHP 8.0 instead.